pkg/bindings/network/network.go

```diff
@@ -50,6 +50,25 @@ func CreateWithOptions(ctx context.Context, network *types.Network, extraCreateO
 	return report, response.Process(&report)
 }
 
+// Updates an existing netavark network config
+func Update(ctx context.Context, netName string, options entities.NetworkUpdateOptions) error {
```

Comment:
please do not use entities in bindings parameters, there is a well established pattern of using its own types to make it more independent, see `ListOptions` or `RemoveOptions` for example

Reply:
also this accept the net name or ID, make sure to call the variable appropriately